<launch>
	<node name="cepheus_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="$(arg controllers_to_spawn)"/>
</launch>
